### ISSUE TITLE
Common namespace for browser and mobile attributes

### ISSUE THREAD

**Issue by scheler** [open-telemetry/opentelemetry-specification#3351]
**What are you trying to achieve?**

This is with reference to this comment  https://github.com/open-telemetry/opentelemetry-specification/pull/3287#discussion_r1135909041 in a PR that is adding a few semantic conventions to browser namespace. There are some attributes that are common to both browser and mobile app instrumentations, and we were wondering what namespace to use if we have to use one common namespace for attributes that are relevant to both these domains (eg., session id, visitor id). Would `client` be appropriate?  

cc: @MSNev  @martinkuba  @breedx-splk @bryce-b

**Comment by reyang** [comment]
I don't see why session or visitor/user id should be client specific (one can imagine that mobile devices, IoT devices or an ATM machine handling similar concept). Maybe just use "user.id" and "user.session"? I suggest that we learn from ECS https://github.com/open-telemetry/oteps/pull/222 and try to align as much as possible.

**Comment by scheler** [comment]
@reyang  it doesn't look like `user` namespace exists currently. There is `enduser`, however, so did you intend to suggest `enduser.id` and `enduser.session` instead?

We discussed the following in the Client Instrumentation SIG today - 

- visitor_id and impression_id fit into `user` or `enduser` whichever we want to keep.
- session_id fits into `client` namespace, if that is allowed to be introduced.

I looked into ECS's `client` namespace - it overlaps with `net` namespace in OTel. If there is a plan to move the networking client aspects to `client` namespace, then we could use it for having `session_id` as well.

cc: @jsuereth
